packages/wonder-blocks-dropdown/components/multi-select-menu.js

```diff
@@ -164,6 +164,31 @@ export default class MultiSelectMenu extends React.Component<Props, State> {
         }
     }
 
+    getMenuText() {
+        const {items, placeholder, selectItemType, selectedValues} = this.props;
+        // If there is nothing selected, use the placeholder if it exists
```

Comment:
🛑 Could you add a comment here with the string "i18n" so that we know to come back later and annotate this so that it can be translated, potentially?